File: models/company.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import declarative_base

base = declarative_base()

class company(base):
    __tablename__ = "company"
    id = Column(Integer, primary_key=True, index=True)
    name = Column(String, nullable=False, index=True)
    email = Column(String, nullable=True)
    phone = Column(String, nullable=True)
    website = Column(String, nullable=True)
    description = Column(String, nullable=True)

File: talentspark/src/types/Company.ts
export interface Job {
  id: number;
  title: string;
  salary: number;
  description?: string;
  company_id: number;
}

export interface Company {
  id: number;
  name: string;
  email: string;
  phone: string;
  location: string;
  jobs: Job[];
}